ruby/red-arrow-cuda/red-arrow-cuda.gemspec

```
@@ -48,4 +48,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("bundler")
   spec.add_development_dependency("rake")
   spec.add_development_dependency("test-unit")
+  spec.required_ruby_version = Gem::Requirement.new("=> 2.6")
```

This is a invalid syntax. `>=` is correct:

```suggestion
  spec.required_ruby_version = Gem::Requirement.new(">= 2.6")
```

---

@kou  Ok, will create new issue and add support for 2.5.  Is this a priority for 6.0.1 ?

---

It's not needed for 6.0.1 because this is only test issue.